ui-tests/cypress/integration/content/assets.spec.js

```diff
@@ -51,21 +60,24 @@ describe('Assets', () => {
     describe([Tag.GTS], 'Delete asset', () => {
 
       it('Delete asset', () => {
-        currentPage = openAssetsPage();
-        cy.wait(2000);
-
-        cy.get('@assetToBeDeleted')
-          .then(assetToBeDeleted => currentPage.getContent().getKebabMenu(assetToBeDeleted.id).open().clickDelete());
-        cy.wait(1000);
-        currentPage.getDialog().confirm();
-
-        cy.validateToast(currentPage);
-
-        cy.wrap(null).as('assetToBeDeleted');
+        openAssetsPage()
+            .then(page => {
+              cy.get('@assetToBeDeleted')
+                .then(() => page.getContent().getKebabMenu().openDropdown().clickDelete())
```

Comment:
On which entry is it working? You've not used the `@assetToBeDeleted` value

Reply:
The only way I have found to be able to access the inserted resource dropdown is by using the name of the image.
Unfortunately, I found no other codes to refer to. 
It is true that the cy.get on alias is absolutely redundant, (I think I've dragged it in from previous refinement iterations)

Reply:
But you are not using the name of the resource here, how is it identifying the element?